packages/react/src/components/participant/ParticipantTile.tsx

```diff
@@ -140,13 +141,21 @@ export const ParticipantTile = ({
       <ParticipantContextIfNeeded participant={p}>
         {children ?? (
           <>
-            {/** TODO remove MediaTrack in favor of the equivalent Audio/Video Track. need to figure out how to differentiate here */}
-            <MediaTrack
-              source={source}
-              publication={publication}
-              participant={participant}
-              onSubscriptionStatusChanged={handleSubscribe}
-            />
+            {source === Track.Source.Camera || source === Track.Source.ScreenShare ? (
```

Comment:
unfortunately this won't handle all cases. 
If we have a publication present we should be using `publication.kind` to determine the right kind of track. 

This also poses the question whether we want to do `track: TrackReferenceOrPlaceholder` instead of spreading the arguments out. This has some ugly side-effects (like implicit context usage will get harder for users), so I'm not totally convinced

Reply:
I agree, I think it still treats it like the old logic. If it was not a `camera` or `screen_share` source, it would always treat it like an audio track.

Reply:
I think adding `publication?.kind === 'video'` would be a rather low hanging fruit?